planter/README.md

```diff
@@ -20,7 +20,8 @@ Then from `$GOPATH/src/k8s.io/kubernetes/` run:
 Planter repects the following environment variables:
 
  - `TAG`: the planter image tag, this will default to the current stable version
- used to build kubernetes, but you may override it with EG `TAG=0.6.1-1`
+ used to build kubernetes, but you may override it with EG `TAG=0.6.1`
+  - These should now match bazel release versions eg `0.8.0rc2`
```

Comment:
I think you need another space to indent this a level?

Reply:
done.